src/jsep.js

```diff
@@ -783,4 +869,42 @@ jsep.removeAllLiterals = function() {
 	return this;
 };
 
+// ternary support as a plugin:
```

Comment:
This should be in a separate file, just like the other plugins. We can include it by default (for compatibility, though I think eventually we should move away from that), but plugins should be separate.